frontend/Chat.tsx

```diff
@@ -35,7 +35,7 @@ const Chat: Component<{
       <Show when={summary()}>
         {(summary) => <Badge>Summary: {summary}</Badge>}
       </Show>
-      <Box width="53vw" maxHeight="36vh" overflow="scroll">
+      <Box overflow="auto">
```

Comment:
This whole box scrolls now which isn't so pretty. It'd be nicer the top part of the box with the selectors and such was fixed and the rest did scroll.

Reply:
I tried that once I believe but I could not get that right. Might have a look into that later again